mkosi.conf.d/20-centos-fedora.conf

```diff
@@ -0,0 +1,7 @@
+[Match]
+Distribution=|centos
+Distribution=|fedora
+
+[Content]
+Packages=
+        systemd-resolved
```

Comment:
This should go into 30-centos-fedora/mkosi.conf. That's the shared centos/fedora config flie now

Reply:
Ah, okay :sweat_smile: